spec/models/gamesession_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
-RSpec.describe Gamesession, type: :model do
-  # pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe GameSession, type: :model do
```

Comment:
My only suggestion here might be to use context/describe blocks to organise these tests cases a bit.

We have one spec which is on the class `defined? GameSession::WORDS`

The three others are about `create`

I'd suggest two blocks:
```
describe 'class constants' do
  it 'should have an array list to select from' do
    defined?(described_class::WORDS) # using described_class. Much of a muchness really
  end
end

describe '.create' do
  subject { GameSession.create }

  it 'word_to_guess should be assigned' do
    expect(subject.word_to_guess).to_not be nil # Using `be nil` - again, just opinionated rspec
  end

  it 'should select word_to_guess from the words array' do
    expect(GameSession::WORDS).to include(subject.word_to_guess) 
  end

  it 'should not have any user guesses to check' do
    expect(subject.guesses).to be nil
  end
end
```

Reply:
btw, this will work evenif you leave the top level `subject { GameSession.new }`. The most recent one will take precedence.